lib/composer.js

```diff
@@ -14,11 +15,16 @@ import MagicParser from './parsers/magic-parser'
 export default class Composer extends Disposable {
   disposables = new CompositeDisposable()
   cachedBuildStates = new Map()
+  cachedDicy = new Map()
 
   constructor () {
     super(() => this.disposables.dispose())
     this.disposables.add(atom.config.onDidChange('latex', () => {
       this.rebuildCompleted = new Set()
+      const dicy = this.cachedDicy.values().next().value
+      if (dicy) {
+        dicy.updateOptions(this.getDicyOptions(), true)
+      }
```

Comment:
What exactly is this the purpose of this section? Seems like this is somewhat non-deterministic. If you only grab the first instance, and update the options on that one, the rest of them are left untouched right? And whatever is first, is entirely dependent upon the what file you build first?